### DISCUSSION TITLE
source.exclude configuration does not take effect

### DISCUSSION THREAD

**Discussion by JonasVon** [web-infra-dev/rsbuild#3152] [General]
文档：
<img width="1030" alt="image" src="https://github.com/user-attachments/assets/581a9d30-6d38-426a-a28c-fcade591ca53">

配置：
<img width="967" alt="image" src="https://github.com/user-attachments/assets/411aa190-3e8c-4bfd-a686-46b5d83fe2d8">

启动后报错：
<img width="1092" alt="image" src="https://github.com/user-attachments/assets/b28694bd-7e31-4acb-a6f9-9e8dab96c12d">

.grammar是一种自定义的语法扩展文件，在rsbuild中被当作js文件解析了，请教大佬们如何让rsbuild忽略这种文件🙏

**Comment by chenjiahan** [comment]
You need to configure Rspack rules to handle the `*.grammar` files.

**Reply by JonasVon** [reply]
怎么在rsbuild中设置rspack的配置，请教下大佬

**Reply by chenjiahan** [reply]
You can refer to this document: https://rsbuild.dev/guide/basic/configure-rspack

**Reply by JonasVon** [reply]
OK，thank you very much
